README.md

```diff
@@ -33,29 +33,30 @@ NOTE: The default configuration will build 64-bit binaries for maximum security
+**IMPORTANT**: Currently, the `MAX_PATH` path length restriction (which is 260 characters by default) must be lifted in for our Python build scripts. This can be lifted in Windows 10 (v1607 or newer) with the official installer for Python 3.6 or newer (you will see a button at the end of installation to do this). See [Issue #345](https://github.com/Eloston/ungoogled-chromium/issues/345) for other methods for older Windows versions.
 
 1. Setup the following:
-
-    * 7-zip
-    * Python 3.6 - 3.9 or 3.10.2+ (for build and packaging scripts used below)
-        * At the end of the Python installer, click the button to lift the `MAX_PATH` length restriction.
-        * Check that your `PATH` does not contain the `python3` wrapper shipped by Windows, as it will only promt you to install Python from the Microsoft Store and exit. See [this question on stackoverflow.com](https://stackoverflow.com/questions/57485491/python-python3-executes-in-command-prompt-but-does-not-run-correctly)
+    * 7-Zip
+    * Python 3.6 - 3.9 or 3.10.2+ (for build and packaging scripts used below); Python 3.11 and above is not supported.
+    * If you don't plan on using the Microsoft Store version of Python:
+    	* Check "Add python.exe to PATH" before install.	
+        * Check that your `PATH` does not contain the `python3` wrapper shipped by Windows, as it will only prompt you to install Python from the Microsoft Store and exit. See [this question on stackoverflow.com](https://stackoverflow.com/questions/57485491/python-python3-executes-in-command-prompt-but-does-not-run-correctly)
+        * Ensure that your Python directory either has a copy of Python named "python3.exe" or a symlink linking to the Python executable.
+   * Make sure to lift the `MAX_PATH` length restriction, either by clicking the button at the end of the Python installer or by [following these instructions](https://learn.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation?tabs=registry#:~:text=Enable,Later).
     * Git (to fetch all required ungoogled-chromium scripts)
         * During setup, make sure "Git from the command line and also from 3rd-party software" is selected. This is usually the recommended option.
 
 ### Building
 
-NOTE: The commands below assume the `py` command was installed by Python 3 into `PATH`. If this is not the case, then substitute it with `python3`.
-
 Run in `cmd.exe` (as administrator):
 
 ```cmd
 git clone --recurse-submodules https://github.com/ungoogled-software/ungoogled-chromium-windows.git
+cd ungoogled-chromium-windows
 # Replace TAG_OR_BRANCH_HERE with a tag or branch name
 git checkout --recurse-submodules TAG_OR_BRANCH_HERE
-py build.py
-py package.py
+python3 build.py
+python3 package.py
```

Comment:
Using `python` instead of `py` or `python3` directly might also work. In this case, we would not need to create `python3` in the first place.

Reply:
It does not work, as Chromium [only looks for `python3`](https://chromium.googlesource.com/chromium/src.git/+/refs/heads/main/.gn)

Reply:
I'm not sure about that, but I am nowhere near my build setup, so I can't verify that. Changing from `py` to `python3` however should not change the behaviour of the scripts. Have you verified that?

Reply:
It does not. It just ensures that there is a `python3` that Chromium will hopefully be able to find.

Reply:
I'm sorry, my question was not precise enough. Have you verified that the proposed changes do work?

Reply:
[Yes](https://cdn.discordapp.com/attachments/745509182559748237/1123685614554976407/Screenshot_2023-06-28_081831.png).

[full console output.txt](https://github.com/ungoogled-software/ungoogled-chromium-windows/files/11899064/full.console.output.txt)